hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -133,9 +133,6 @@ public void start() {
         .setDaemon(false)
         .setNameFormat(THREAD_NAME + " - %d")
         .setUncaughtExceptionHandler((Thread t, Throwable ex) -> {
-          // gracefully shutdown SCM.
-          scmContext.getScm().stop();
-
```

Comment:
I think this stop should be called to stop all the SCM services. 

It is the terminate which needs to be moved to StorageContainerManager after all the other processes are stopped.